spiffworkflow-backend/src/spiffworkflow_backend/services/process_instance_processor.py

```diff
@@ -1900,16 +1900,14 @@ def terminate(self) -> None:
         db.session.add(self.process_instance_model)
         db.session.commit()
 
-    @classmethod
-    def suspend(cls, process_instance: ProcessInstanceModel) -> None:
+    def suspend(self) -> None:
```

Comment:
Any reason to not keep these as classmethods and change the lock method to a classmethod. It doesn''t seem like the lock method needs anything from the processor except for the process instance anymore which can be passed in.

Reply:
Good point - sorry I missed this before hitting merge. Most of the callers (granted most are in tests) already have the instance, but yes. Really want I want to move to in places like this are a `with locked_*():` type of call so we don't necessarily need any instance and also don't need all the try/catch/except stuff. Will put that down on the todos.